packages/contracts/src/contracts/PublicLock/PublicLockV14.sol

```diff
@@ -2842,7 +2842,7 @@ interface IUnlock {
    * @notice This function can be called by anyone (not only the contract owner) as a way to ensure decentralization.
    * @param token the address of the token (zero address for native) to swap and burn
    * @param amount the amount of tokens to swap and burn
-   * @param poolFee the poolFee of the token <> WETH/Wrapped Native asset to  use for the swap
```

Comment:
the `<>` characters were interpreted as a html tag by the docgen, so had to replace it